--- xpaint-2.9.8/configure~	2010-11-07 14:40:41.000000000 +0100
+++ xpaint-2.9.8/configure	2012-01-27 11:47:26.911805498 +0100
@@ -104,8 +104,8 @@
    ln -sf ./Xaw3dxft xaw_incdir
    echo "XAWLIB_DEFINES = -DXAW3D -DXAW3DG -DXAW3DXFT"   
    echo "XAWLIB_DEFINES = -DXAW3D -DXAW3DG -DXAW3DXFT" > Local.xawdefs
-   echo "SYS_LIBRARIES = -lXaw3dxft -lXt -L. -L./Xaw3dxft"
-   echo "SYS_LIBRARIES = -lXaw3dxft -lXt -L. -L./Xaw3dxft" >> Local.xawdefs
+   echo "SYS_LIBRARIES = -lXaw3dxft -lXt -lfontconfig -L. -L./Xaw3dxft"
+   echo "SYS_LIBRARIES = -lXaw3dxft -lXt -lfontconfig -L. -L./Xaw3dxft" >> Local.xawdefs
    echo ""
    cd Xaw3dxft ; xmkmf ; make Xaw3dP.h ; cd .. ;
    xmkmf -a
