--- xpaint-3.1.4/util/Makefile.orig	2020-08-09 19:20:37.000000000 +0200
+++ xpaint-3.1.4/util/Makefile	2025-09-27 01:19:59.253326383 +0200
@@ -10,7 +10,7 @@
 endif
 
 pdfconcat:
-	$(CC) $(CFLAGS) -O3 -s -DNDEBUG=1 -DNO_CONFIG=1 -ansi -pedantic -Wunused -Wall -W -Wnested-externs -Winline -Wpointer-arith -Wbad-function-cast -Wcast-qual -Wmissing-prototypes -Wmissing-declarations pdfconcat.c -o pdfconcat
+	$(CC) $(CFLAGS) -DNDEBUG=1 -DNO_CONFIG=1 -ansi -pedantic -Wunused -Wall -W -Wnested-externs -Winline -Wpointer-arith -Wbad-function-cast -Wcast-qual -Wmissing-prototypes -Wmissing-declarations pdfconcat.c -o pdfconcat
 
 ppmtops:
 	$(CC) $(CFLAGS) ppmtops.c -o ppmtops
